app/services/proofing/aamva/request/verification_request.rb

```diff
@@ -46,9 +72,21 @@ def verification_url
           config.verification_url || DEFAULT_VERIFICATION_URL
         end
 
+        # The requested attributes in the applicant PII hash. Values are:
+        # - +1+ - value present
+        # - +0+ - field is required, but value was blank
```

Comment:
nit: maybe `:present` and `:blank` or similar. Cloudwatch logs should include `1` or `0`, but for an internal API I think more descriptive values would be nice.

Reply:
Done!